pvlib/tests/test_modelchain.py

```diff
@@ -447,29 +447,52 @@ def acdc(mc):
     mc.ac = mc.dc
 
 
+# TODO in v0.9: remove 'snlinverter', 'adrinverter' since these values for
+# ModelChain.ac_model are superseded by 'sandia' and 'adr'
 @pytest.mark.parametrize('ac_model', [
-    'snlinverter', pytest.param('adrinverter', marks=requires_scipy),
+    'sandia', 'snlinverter', pytest.param('adrinverter', marks=requires_scipy),
+    pytest.param('adr', marks=requires_scipy),
```

Comment:
remove the old keywords from this test to avoid deprecation warnings in the logs

https://dev.azure.com/solararbiter/pvlib%20python/_build/results?buildId=3731&view=logs&j=fc432b8b-e2e3-594e-d8b1-15597b6c1d62&t=309866e1-2cf4-5f00-3d0a-999fc3a0f279&l=235